Source/Svg.csproj

```diff
@@ -144,6 +144,62 @@ Many performance improvements by using SourceGenerators instad of reflection.
   <PropertyGroup Condition="'$(TargetFramework)' != 'net452'">
     <DefineConstants>$(DefineConstants);USE_SOURCE_GENERATORS</DefineConstants>
   </PropertyGroup>
+    
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Is this property required for each ?

Reply:
I suppose so? That was auto-generated so I'm not 100% sure but please let me know what you think the better approach would be.